packages/runner/src/types/tasks.ts

```diff
@@ -259,6 +260,11 @@ export interface TestContext {
    */
   task: Readonly<Test>
 
+  /**
+   * Diff options
+   */
+  diff?: DiffOptions
```

Comment:
Why is it on a task?

Reply:
My mistake, just realized we can access it by `runner.config`